### DISCUSSION TITLE
DIY Haikubox style project

### DISCUSSION THREAD

**Discussion by frrobert2** [birdnet-team/BirdNET-Analyzer#360] [Q&A]
Good afternoon,

I have a DIY project and I have a couple of questions.  The project is just for fun.  I am not looking to report the information anywhere.  I know the results are  are questionable at this point.

- I have an POE Security Camera with a microphone that monitors our bird feeder.  I put it up several years ago so my wife and I could watch the birds remotely.  The camera can run as an RTSP server and supports both audio and video.
- I installed BirdNet-Analyzer on my Linux Box.
- I wrote a bash script with a loop that takes a five second stream from the RTSP server saves it a file which is then passed to BirdNet-Analyzer. So every five seconds BirdNetAnalyzer gets a new five second clip to process.  All the results are appended to a text file.

Every day, through out the day I send a list of birds heard for that day to my blog page.

https://www.frrobert.com/blog/birdsatmckinley

- Current settings for BirdNet-Analyzer and the bash script as follows:
     -  I have Longitude and Latitude set as command line options
     -  Currently I am doing five second clips
     -  I am only savings results with a confidence higher than 0.500

Here are my questions:

- Should I use species_list.txt rather than Latitude and Longitude?  I do have several false positives every day. 
-  From reading the documentation it looks like BirdNet-Analyzer processes three second clips.  Should I change my recording time from five seconds to three seconds or a multiple of three?
- Is there a recommended confidence level I should use?

Thanks,

Robert

**Comment by tphakala** [comment]
Optimal recording lengths are multiples of 3 second, so 6 second would be better than your current 5 second.
If you are seeing lots of false positives I'd increase confidence level, maybe 0.8 (80%) would be a good compromise.

But is there a reason you are not using actual realtime detection application like my https://github.com/tphakala/birdnet-go or https://github.com/mcguirepr89/BirdNET-Pi? Nothing wrong with DIY but just wanted to let you know of alternatives.

**Reply by frrobert2** [reply]
Thank you for the feedback.  I know about BirdNet-PI but I didn't know about birdnet-go.  I am running BirdNet on a 64-bit Intel based desktop, so I didn't look at Birdnet-PI but it looks like birdnet-go will run on on my machine so I may have to give it a try.

**Reply by alexbelgium** [reply]
Hi, the new fork of BirdNET pi whichsupports x64 is here : https://github.com/Nachtzuster/BirdNET-Pi. It's python analysis code was optimized vs the initial b-pi which reduced resources usage also. 

I've set homeassistant addons for both birdnet-go and birdnet-pi which allows testing them in parallel using the dockerized HA system (if you use this app). Both run fine (even in parallel) on my rpi, so would run perfectly on a intel desktop.

**Reply by kahst** [reply]
Do you think it is time to replace Patrick's original BirdNET-Pi version with a new fork? Should we update the showroom link and setup instructions?

**Reply by alexbelgium** [reply]
Hi, I would highly recommend it. The new birdnet-pi fork https://github.com/Nachtzuster/BirdNET-Pi supports more devices (incl. x64 and homeassistant based) ; more functions (both in UI and analysis) ; and more importantly is actively supported. With proper credits to Patrick of course for having setup the initial birdnet pi!

To be noted that in parallel tphakala is also developing an alternative to birdnet-pi here : https://github.com/tphakala/birdnet-go

**Comment by DaveBKC** [comment]
Agreed. As someone who recently returned to this project after sponsoring Patrick’s efforts, it was a wee bit confusing. On Jul 9, 2024, at 8:19 AM, Alexandre ***@***.***> wrote:﻿
Hi, I would highly recommend it. The new birdnet-pi fork https://github.com/Nachtzuster/BirdNET-Pi supports more devices (incl. x64 and homeassistant based) ; more functions (both in UI and analysis) ; and more importantly is actively supported. With proper credits to Patrick of course for having setup the initial birdnet pi!
